src/main/java/Listener/CountingEnforcement.java

```diff
@@ -21,15 +24,15 @@ public CountingEnforcement (DiscordApi api) {
     }
     @SneakyThrows
     private void countCheck(TextChannel channel, Message message, MessageAuthor author) {
-        if (author.getIdAsString().equals(BotInfo.getBotId())) {
-            return;
-        }
-        if (!channel.getIdAsString().equals("519563130603307018")) {
+        // ignore messages not in this one hardcoded channel or from the bot
+        if (!channel.getIdAsString().equals("519563130603307018") ||
+            author.getIdAsString().equals(BotInfo.getBotId())) {
```

Comment:
why have two short circuit checks when you can have one and place the far more likely case first?

Reply:
I'm a little stupid sometimes